File: messages/src/main/java/com/thundashop/zauiactivity/dto/BookingZauiActivityItem.java
package com.thundashop.zauiactivity.dto;

import java.util.List;

import com.thundashop.core.pmsmanager.PmsBookingAddonItem;

import lombok.Getter;
import lombok.Setter;
import lombok.AccessLevel;

@Getter 
@Setter
public class BookingZauiActivityItem extends PmsBookingAddonItem {
    @Setter(AccessLevel.NONE)
    private String id = java.util.UUID.randomUUID().toString();

    private String zauiActivityId;
    private Integer octoProductId;
    private String optionTitle;
    private String optionId;
    private String availabilityId;
    private List<Unit> units;
    private double unpaidAmount;
    private String notes;
    private String localDateTimeStart;
    private String localDateTimeEnd;
    private Integer supplierId;
    private String supplierName;
    private OctoBooking octoBooking;
}

File: com.getshop.client/events/zauiactivity/dto/BookingZauiActivityItem.php
<?php
class zauiactivity_dto_BookingZauiActivityItem extends core_pmsmanager_PmsBookingAddonItem  {
	/** @var String */
	public $id;

	/** @var String */
	public $zauiActivityId;

	/** @var String */
	public $octoProductId;

	/** @var String */
	public $optionTitle;

	/** @var String */
	public $optionId;

	/** @var String */
	public $availabilityId;

	/** @var zauiactivity_dto_Unit[] */
	public $units;

	/** @var zauiactivity_dto_Pricing */
	public $pricing;

	/** @var String */
	public $unpaidAmount;

	/** @var String */
	public $notes;

	/** @var String */
	public $localDateTimeStart;

	/** @var String */
	public $localDateTimeEnd;

	/** @var String */
	public $supplierId;

	/** @var String */
	public $supplierName;

	/** @var zauiactivity_dto_OctoBooking */
	public $octoBooking;

}
?>